mbed_devices/_internal/resolve_target.py

```diff
@@ -50,7 +43,10 @@ def _build_target_resolver(candidate: CandidateDevice) -> Callable:
         online_id = _extract_online_id(all_files_contents)
         return functools.partial(get_target_by_online_id, slug=online_id.device_slug, target_type=online_id.device_type)
     except OnlineIdNotFound:
-        raise UnableToBuildResolver
+        pass
```

Comment:
I'd like an explicit comment as to why we `pass` here; for future me when I forget what this is doing.